types/V1.mli

```diff
@@ -1,6 +1,6 @@
 (*
- * Copyright (c) 2011-2014 Anil Madhavapeddy <anil@recoil.org>
- * Copyright (c) 2013      Thomas Gazagnaire <thomas@gazagnaire.org>
+ * Copyright (c) 2011-2015 Anil Madhavapeddy <anil@recoil.org>
+ * Copyright (c) 2013-2015 Thomas Gazagnaire <thomas@gazagnaire.org>
```

Comment:
IANAL but I personally stopped updating copyrights date in my sources. IIUC basically the first date tells when you claim the copyright for the first time and that should be enough, it will be protected, at least in [Switzerland](http://www.admin.ch/opc/en/classified-compilation/19920251/index.html#a29), for 50 years after your death (fun that it's shorter by 20 years for computer programs...). So unless you care about a few years more for your descendants I don't think it's very useful. I would love to get some confirmation by a legal person though.